Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM python:3.7-slim as cloudmapper
+
+ARG git_version
+
+WORKDIR /opt/manheim_cloudmapper
+
+RUN apt-get update -y
+RUN apt-get -y install build-essential git autoconf automake libtool python3.7-dev python3-tk jq awscli python3-pip
+RUN apt-get install -y bash
+
+RUN git clone --branch 2.7.2 https://github.com/duo-labs/cloudmapper.git /opt/manheim_cloudmapper
+
+RUN mkdir /opt/manheim_cloudmapper/port_check
+COPY manheim_cloudmapper/* /opt/manheim_cloudmapper/
+
+COPY manheim_cloudmapper/port_check/ /opt/manheim_cloudmapper/port_check/
+COPY manheim_cloudmapper/ses/ /opt/manheim_cloudmapper/ses/
+
+RUN chmod +x /opt/manheim_cloudmapper/cloudmapper.sh
+
+RUN pip install pipenv
+RUN pipenv install premailer --skip-lock
+RUN pipenv install --skip-lock
+
+RUN bash
```

Comment:
So I had `RUN bash` in here because I was basing my Dockerfile from the official Dockerfile for cloudmapper. (https://github.com/duo-labs/cloudmapper/blob/master/Dockerfile). I saw they have `RUN bash` as the last line and was thinking it may have something to do with the way cloudmapper works? But I agree that it should be taken out. 

On the CMD or ENTRYPOINT comment. I agree that we should add that, however what happens during the docker test when we enter the container and attempt to run cloudmapper.sh?